readme.md

```diff
@@ -185,44 +183,6 @@ Use them in your terminal like: `npm run <script>` :
 | dist:designsystem | Builds a distribution folder of the Designsystem library                                                                                               |
 | transpile:tools   | Transpiles tools, required to produce library distribution (this is done as a `post-install` hook, but may have value if altering tool implementation) |
 
-## Polyfills
-
-Some features of Kirby requires polyfills to ensure compability across all major browsers _(e.g. the `ResizeObserverService` used by the automagic sizing feature of the `Kirby Card` component)_.
-
-To enable the polyfill, you register a _polyfill loader_ that checks whether the polyfill is needed or the feature is already supported by the browser (and can skip requesting the polyfill).  
-To use the sizing feature of Kirby Card across all major browsers, you must copy additional files from the Kirby package to your output folder.  
-Add the following to `build > options > assets` in `angular.json` :
-
-```json
-{
-  ...
-  "build": {
-    "options": {
-      "assets": [
-        ...,
-        {
```

Comment:
Isn't it a breaking change if people reference this folder, and we remove it after this PR and the IntersectionObserver polyfill PR are merged? 
Should we write in migrations.md, that people should remove this part of the `angular.json` config?

If we should add it to migrations.md, then I will do it when I'm updating it with the stuff we already talked about for v8 :)

Reply:
You are probably right! Our own cookbook failed silently, but that might just be due to me not properly restarting the dev server, now that I think about it. 

Lets add it to be sure!